generator/template.go

```diff
@@ -558,16 +558,18 @@ func tsType(r *registry.Registry, fieldType data.Type) string {
 func mapWellKnownType(protoType string) string {
 	switch protoType {
 	case ".google.protobuf.BoolValue":
-		return "boolean | undefined"
+		return "boolean | null"
 	case ".google.protobuf.StringValue":
-		return "string | undefined"
+		return "string | null"
 	case ".google.protobuf.DoubleValue",
 		".google.protobuf.FloatValue",
 		".google.protobuf.Int32Value",
 		".google.protobuf.Int64Value",
 		".google.protobuf.UInt32Value",
 		".google.protobuf.UInt64Value":
-		return "number | undefined"
+		return "number | null"
+	case ".google.protobuf.ListValue":
+		return "T[]"
```

Comment:
👍 This change makes total sense to me. I wrote that function from scratch in a previous PR, and I think I just made a wrong assumption that undefined would be serialized for well known wrapper types when the wrapper was not set (nil) in go code. Looking at the currently generated TS code, I think this will work great.